### DISCUSSION TITLE
Lasergrbl Windows 11 installation issue

### DISCUSSION THREAD

**Discussion by DonJ56** [arkypita/LaserGRBL#1687] [Q&A]
Trying to get Lasergrbl to install on a Win11 laptop. Anyone have any success?

**Comment by arkypita** [comment]
LaserGRBL works perfectly with windows 11
Did you have any problems? Explains better.

**Reply by Chucka87** [reply]
I am also having an issue trying to load LaserGRBL on a windows 11 laptop.  When i select the "download" button i get a message that states----"install(2)exe isn't commonly downloaded.  Make sure you trust it before opening"   So i clicked to keep the file and then i continued to say it is safe to Microsoft.  However after a few attempts I cannot get the exe file.  I do get an "unconfirmed " download file but cannot open it or do anything with it.  Can someone give me step by step instructions how to get the laserGRBL downloaded and running on my windows 11 laptop

**Reply by bjbutter** [reply]
where do you go if you are not running in S Mode? I still can not download program

**Reply by Tribo-2dot0** [reply]
> where do you go if you are not running in S Mode? I still can not download program

clique em Mais Ações (...) e escolha 'Manter' e depois 'Mostrar mais' e 'Manter mesmo assim'... aí o .exe estará disponível no local de download.

**Comment by Tribo-2dot0** [comment]
Disable Windows 11 S Mode.

**Reply by Chucka87** [reply]
I was worried about doing that but then did disable S mode. The program loaded fine.   Thanks much for the help
